workflow/common/common.go

```diff
@@ -79,6 +79,8 @@ const (
 	LabelKeyWorkflowArchivingStatus = workflow.WorkflowFullName + "/workflow-archiving-status"
 	// LabelKeyWorkflow is the pod metadata label to indicate the associated workflow name
 	LabelKeyWorkflow = workflow.WorkflowFullName + "/workflow"
+	// LabelKeyTemplate is the pod metadata label to indicate the associated template name
+	LabelKeyTemplate = workflow.WorkflowFullName + "/template"
```

Comment:
Hmm I thought the issue meant `WorkflowTemplate`, for which the label key already exists on line 92 below.

`template` names are also not necessarily unique, unlike a `Workflow`, `CronWorkflow`, or `WorkflowTemplate` name. Though if the Pod has all the labels you can uniquely identify it from a combination

Reply:
Thanks, I added WorkflowTemplate. Maybe Template also has some application scenarios. For example, what we are doing is automatically adjusting the CPU and memory for the new workflow based on the resource usage of the past workflow. It needs to correspond to the resource usage of a template and pod. If there is a label of the template, it will Much more convenient.